backends/backends.c

```diff
@@ -900,43 +1043,77 @@ void *backends_main(void *ptr) {
 
 #if HAVE_MONGOC
         if(do_mongodb) {
+            for(int i = 0; i < MONGODB_THREADS_NUMBER; i++) {
```

Comment:
Some old C compilers  does not accept declaration of variables inside a `for`, case my memory is correct, this was only brought to ANSI C with C 99, please, put the declaration of the variable out side the loop.

Reply:
At least 7 of C99 features are already in use in the Netdata code. This particular feature has not been used yet, so I'll fix it. Let us discuss the matter on the meetup.

Reply:
All right, my worry is with people using old Linux version that won't have this option enabled by default.